ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/RewardsActionsComposable.kt

```diff
@@ -58,10 +57,13 @@ fun RewardsActions(
     ) {
       onClickGiftCard()
     }
-    Spacer(
-      modifier = Modifier
-        .width(16.dp)
-    )
+    ActionCard(
+      image = R.drawable.ic_eskills,
+      title = R.string.rewards_eskills_card_title,
+      description = R.string.rewards_eskills_card_body
+    ) {
+      onClickEskills()
```

Comment:
Since you're refactoring the cards, do you mind reorganizing the ActionCard calls, i think it would be better if all arguments were inside the parentheses "(...)". Using composable components, only other composable items stay inside braces "{...}", like the content of a `Row`, `Button`, etc. onClick stays inside parentheses "(...)" but the other composables inside braces "{...}".

Reply:
good point, I already changed it